mav_msgs/include/mav_msgs/conversions.h

```diff
@@ -19,6 +19,10 @@
 #ifndef MAV_MSGS_CONVERSIONS_H
 #define MAV_MSGS_CONVERSIONS_H
 
+#include <Eigen/StdVector>
```

Comment:
Eigen should go down into other group. https://github.com/ethz-asl/multiagent_mapping/wiki/Header-include-guide